src/modules/hotkeys/i18n/de_DE.root.json

```diff
@@ -74,14 +74,30 @@
             "title": "Hauptfenster"
         },
         "mission": {
+            "alliance": {
+                "focus": "Fokussiere Rückmeldungsfeld",
+                "title": "Verband",
+                "toggle": "Wechsle 'Im Verbandschat posten'"
+            },
+            "arr": {
+                "next": "Nächster Tab",
+                "previous": "Vorheriger Tab",
+                "title": "AAO"
+            },
             "sorted": {
                 "alert_next": "Alarmieren und weiter",
                 "alert_share_next": "Alarmieren, Teilen und weiter",
                 "next": "nächster Einsatz",
                 "prev": "vorheriger Einsatz",
                 "title": "Einsatzsortierung"
             },
-            "title": "Einsatzfenster"
+            "title": "Einsatzfenster",
+            "vehicleList": {
+                "loadMissing": "Lade fehlende Fahrzeuge",
```

Comment:
I think I would prefer `Fehlende Fahrzeuge nachladen` but it's up to you

Reply:
Changed in [d7a8d53](https://github.com/LSS-Manager/LSSM-V.4/pull/1717/commits/d7a8d53e6c12024ab560fb108f86293bf001a332)